app/models/resource.py

```diff
@@ -44,19 +45,28 @@ def create_resource(name, description, website):
         return result
 
     @staticmethod
-    def generate_fake(count=10):
+    def generate_fake():
+        # TODO: make sure fake resources have users
+
         """Generate count fake Resources for testing."""
         from faker import Faker
 
         fake = Faker()
 
-        # TODO: make sure fake resources have users and addresses
-        for i in range(count):
+        addresses = Address.query.all()
+        unused_addresses = []
+        for i in addresses:
+            resources = i.resources.all()
+            if len(resources) == 0:
+                unused_addresses.append(i)
+
+        for i in unused_addresses:
             r = Resource(
```

Comment:
does this mean we are guaranteeing that we will have no more than `len(Address.query.all())` in fake resources?